addons/io_hubs_addon/components/components_registry.py

```diff
@@ -113,6 +142,28 @@ def unregister_component(component_class):
         print(f"Component unregistered: {component_class.get_name()}")
 
 
+def load_user_components():
+    global __components_registry
+    for module in get_user_component_definitions():
+        for _, member in inspect.getmembers(module):
+            if inspect.isclass(member) and issubclass(member, HubsComponent) and module.__name__ == member.__module__:
+                if hasattr(module, 'register_module'):
+                    module.register_module()
+                register_component(member)
+                __components_registry[member.get_name()] = member
```

Comment:
Currently, if there are any bugs in one of the user definitions it will break the whole add-on, so I think this section should be wrapped in a try/except block.  Something like this:
```suggestion
                try:
                    if hasattr(module, 'register_module'):
                        module.register_module()
                    register_component(member)
                    __components_registry[member.get_name()] = member
                except Exception:
                    import traceback
                    traceback.print_exc()
```

Reply:
Added